db/migrate/20231218154837_create_costs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :costs, id: :uuid do |t|
+      t.references :claim, null: false, foreign_key: true, type: :uuid
+      t.string :type, null: false
+      t.float :amount, null: false
+      t.float :amount_with_vat
+    end
```

Comment:
did you consider storing the data in a flat structure instead of multiple records? 

I ask as that is what I had originally imagined - it doesn't make a heap of difference here, but would potentially be an easier data structure to use/query in the CW side. :)

Reply:
My preference is to keep is as a many relationship where the type differentiates it so that we don't have to keep extending the table if there are new totals that get added - and we can easily reuse this table if we use it for CRM4/5 etc. However probably true that it's easier to query on the CW side.